docs/Roadmap.md

```diff
@@ -20,4 +20,20 @@ Any individual or organization can create an Issue on the OTP Roadmap using the
 ![choose-issue-type](images/roadmap-5.png)
 7. Fill in the necessary fields, and then click `Create`.
 <br>
-![create](images/roadmap-6.png)
\ No newline at end of file
+![create](images/roadmap-6.png)
+<be>
+
+
+A roadmap issue should follow these principles:
+
+Encourage discussions in the thread, [Gitter](https://app.gitter.im/#/room/#opentripplanner_OpenTripPlanner:gitter.im), PO- or developer meetings. Discussions should be concluded, and the roadmap description should be updated accordingly.
+ - The description must be easy to understand and always updated to the latest conclusion. Click the “Edit” button. 
+ - Description should at least consist of 
+   - What’s the problem we want to solve
+   - Why we want to, or the effect it makes
+   - Linked issues
+ - Description could include a solution
+ - Use LABELS
+Twice a year, POs and developers will sort, time frame, and clean up the issues.
```

Comment:
"Twice a year, product owners (POs) and developers will review and clean up the issues at the PO meetings"